faillint/faillint.go

```diff
@@ -52,15 +52,30 @@ func run(pass *analysis.Pass) (interface{}, error) {
 
 	for _, file := range pass.Files {
 		for _, path := range imports {
-			imp := usesImport(file, path)
+			imp, exact := usesImport(file, path)
```

Comment:
I'm not sure about the "exact" word here. What do you think about "fullMatch" or "exactMatch" ?   Or you reverse it and return the "subMatch" (or "subPath"). Then below you could say `if subMatch { ... }` which I think is easier to read and reason than `if !exact { ... }`

Reply:
Thank you for the comment, I agree!